### ISSUE TITLE
export 'default' (imported as 'delegate') was not found in 'delegate-it' (possible exports: oneEvent)

### ISSUE THREAD

**Issue by mmarton** [fregante/delegate-it#47]
Error in title is shown after upgrading to v6, and using webpack to compile

```js
import delegate from 'delegate-it';

function edit(event) {...}

delegate('.edit', 'click', edit);
```

**Comment by fregante** [comment]
Thank you for the quick report. I'll fix it in a few minutes

**Comment by fregante** [comment]
Fixed in 6.0.1

**Comment by mmarton** [comment]
wow that was fast, thanks
